Makefile

```diff
@@ -60,7 +60,7 @@ release: clean
 	# Now generate the release notes to have them included in the release commit
 	towncrier --yes --version $(UPCOMING_VERSION)
 	# Before we bump the version, make sure that the towncrier-generated docs will build
-	make build-docs
+	$(MAKE) build-docs
```

Comment:
Just curious: Why is this the preferred over just `make build-docs`?

Reply:
On OSes like Solaris, it was a way to use the same make command that the user typed in, which often was ``gmake`` because the Solaris make didn't understand GNU Make extensions.

Reply:
Thanks for filling my knowledge gap!